eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -37,16 +37,17 @@ jobs:
     # Linux musl x64
     - ${{ if eq(parameters.platform, 'Linux_musl_x64') }}:
       - ${{ if eq(parameters.jobParameters.isFullMatrix, false) }}:
-        - (Alpine.311.Amd64.Open)ubuntu.1604.amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.11-helix-bfcd90a-20200123191053
+        - (Alpine.312.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix-20200601195603-e06dc59
       - ${{ if eq(parameters.jobParameters.isFullMatrix, true) }}:
         - (Alpine.39.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.9-helix-bfcd90a-20200123191053
```

Comment:
I think we should also remove Alpine.39 from the test matrix as it is not in the supported platforms for .NET 5.0: https://github.com/dotnet/core/blob/master/release-notes/5.0/5.0-supported-os.md#linux

cc: @terrajobst @danmosemsft